test/system/211-dbus.bats

```diff
@@ -58,22 +58,372 @@ teardown() {
+                         org.freedesktop.systemd1.Manager \
+                         Version)"
+
+  create_distro_container fedora 34 fedora-toolbox-34
+
+  run --keep-empty-lines --separate-stderr "$TOOLBOX" run --distro fedora --release 34 gdbus call \
+                                                                          --system \
+                                                                          --dest org.freedesktop.systemd1 \
+                                                                          --object-path /org/freedesktop/systemd1 \
+                                                                          --method org.freedesktop.DBus.Properties.Get \
+                                                                          org.freedesktop.systemd1.Manager \
+                                                                          Version
+
+  assert_success
+  assert_line --index 0 "$expected_response"
+
+  if check_bats_version 1.10.0; then
+    assert [ ${#lines[@]} -eq 1 ]
+  else
+    assert [ ${#lines[@]} -eq 2 ]
+  fi
+
+  # shellcheck disable=SC2154
+  assert [ ${#stderr_lines[@]} -eq 0 ]
+}
+
+@test "dbus: system bus inside RHEL 8.7" {
+  local expected_response
+  expected_response="$(gdbus call \
+                         --system \
+                         --dest org.freedesktop.systemd1 \
+                         --object-path /org/freedesktop/systemd1 \
+                         --method org.freedesktop.DBus.Properties.Get \
+                         org.freedesktop.systemd1.Manager \
+                         Version)"
+
+  create_distro_container rhel 8.7 rhel-toolbox-8.7
+
+  run --keep-empty-lines --separate-stderr "$TOOLBOX" run --distro rhel --release 8.7 gdbus call \
+                                                                        --system \
+                                                                        --dest org.freedesktop.systemd1 \
+                                                                        --object-path /org/freedesktop/systemd1 \
+                                                                        --method org.freedesktop.DBus.Properties.Get \
+                                                                        org.freedesktop.systemd1.Manager \
+                                                                        Version
+
+  assert_success
+  assert_line --index 0 "$expected_response"
+
+  if check_bats_version 1.10.0; then
+    assert [ ${#lines[@]} -eq 1 ]
+  else
+    assert [ ${#lines[@]} -eq 2 ]
+  fi
+
+  # shellcheck disable=SC2154
+  assert [ ${#stderr_lines[@]} -eq 0 ]
+}
+
+@test "dbus: system bus inside Ubuntu 16.04" {
+  local expected_response
+  expected_response="$(busctl call \
+                        --system org.freedesktop.systemd1 \
+                        /org/freedesktop/systemd1 \
+                        org.freedesktop.DBus.Properties Get "ss" \
+                        org.freedesktop.systemd1.Manager Version)"
```

Comment:
`busctl(1)` has a separate `get-property` command that simplifies fetching the value of D-Bus properties.  eg., this could be re-written as:
```
$ busctl --system get-property org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager Version
s "253.12-1.fc38"
```

Reply:
Done.